src/main/java/org/tdl/vireo/model/repo/impl/UserRepoImpl.java

```diff
@@ -21,9 +26,12 @@ public User create(String email, String firstName, String lastName, AppRole role
         
         newUser = userRepo.save(newUser);
         
+        NamedSearchFilter activeFilter = namedSearchFilterCriteriaRepo.create(newUser);
+        
         newUser.putSetting("id", newUser.getId().toString());
         newUser.putSetting("displayName", newUser.getFirstName() +" "+ newUser.getLastName());
         newUser.putSetting("preferedEmail", newUser.getEmail());
+        newUser.setActiveFilter(activeFilter);
```

Comment:
All users probably should not have an active filter. Most users will be students and will not need one.